shenyu-client/shenyu-client-websocket/shenyu-client-spring-websocket/src/main/java/org/apache/shenyu/client/spring/websocket/init/SpringWebSocketClientEventListener.java

```diff
@@ -93,9 +93,14 @@ public SpringWebSocketClientBeanPostProcessor(final PropertiesConfig clientConfi
     }
 
     @Override
-    public Object postProcessAfterInitialization(@NonNull final Object bean,
-        @NonNull final String beanName) throws BeansException {
+    public void onApplicationEvent(final ContextRefreshedEvent contextRefreshedEvent) {
+        String[] beanNames = contextRefreshedEvent.getApplicationContext().getBeanDefinitionNames();
```

Comment:
After testing I found a problem with this. It will not get the bean when @ShenyuSpringWebSocketClient marked on the method. 
like this:

```
@Component
public class Config {
    @ShenyuSpringWebSocketClient(path = "/org/apache/shenyu/examples/websocket/**", desc = "onlineusers")
    public void handle() {
        return;
    }
}
```

Reply:
I see and I still recommend `contextRefreshedEvent.getApplicationContext().getBeansWithAnnotation(ShenyuSpringWebSocketClient.class);`, but keep line 107. In this way, most useless beans can be filtered out at the first time.

Reply:
ok